### ISSUE TITLE
[FR] Allow to navigate more than 1 previous tab

### ISSUE THREAD

**Issue by noname-oni** [MitchellMarkGeorge/Boomerang#1]
Thank you for the really wanted feature! But when I found your product on PH I thought it will allow navigating much more than 1 previous active tab. Even Hotkey seems more like a hotkey for sequence rather than back-forward.

💖

**Comment by MitchellMarkGeorge** [comment]
Hey @cogscides! Thanks for the request. If you actually go back a few commits, you might notice that is what I originally did. It is defiantely something I am interested in doing but I feel that more people might only need the previous tab in an instant.

**Comment by noname-oni** [comment]
@MitchellMarkGeorge I'm sure that navigating more than 1 step back is more relevant for most people.
Maybe two options? And yeah in the case of multi-step it should allow moving forward-backward in the sequence.

**Comment by MitchellMarkGeorge** [comment]
Having multiple preferences would definitely be a logical choice! One of the original ideas that I had was for users to be able to choose how many previous tabs they want to be stored at a given point in time (eg: only being able to switch to 3 previous tabs at any given period). I greatly appreciate your feedback.

**Comment by noname-oni** [comment]
@MitchellMarkGeorge Yeah, that would be awesome. Thank you for that neat and elegant extension!
